_data/developer.yml

```
@@ -188,6 +188,7 @@ websites:
     tfa:
       - sms
       - totp
+      - u2f
     doc: https://help.compose.com/docs/two-factor-authentication
```

Since the old page is not up to date, why not use the blog entry as the new (interim) doc link?
```suggestion
    doc: https://www.compose.com/articles/introducing-fido-universal-2nd-factor-authentication/
```